wix/windows-runtime.wxs

```diff
@@ -1,44 +1,84 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
-  <Product Id="*"
-           Language="1033"
-           Manufacturer="dt.compnerd.org"
-           Name="Windows Swift Runtime for x86_64"
-           UpgradeCode="bed47738-52ef-4af7-a47b-b81cf238e0c8"
-           Version="0.0.0.0">
-    <Package Comments="Copyright 2019-2020 Saleem Abdulrasool &lt;compnerd@compnerd.org&gt;"
-             Compressed="yes"
+  <Module Id="dt.compnerd.org.SwiftRuntime" Language="1033" Version="0.0.0.0">
+    <Package Id="2cb14917-cbfc-413e-9a02-5be101f8757f"
+             Comments="Copyright 2019-2020 Saleem Abdulrasool &lt;compnerd@compnerd.org&gt;"
              Description="Windows Swift Runtime for x86_64"
              InstallScope="perMachine"
              Manufacturer="dt.compnerd.org" />
 
-    <!--
-      NOTE(compnerd) use the pre-3.8 schema for better compatibility.  If we
-      are guaranteed WIX 3.8+ we could write this as:
-
-      <MediaTemplate EmbedCab="yes" />
-    -->
-    <Media Id="1" Cabinet="runtime.cab" EmbedCab="yes" />
-
-    <?ifdef INCLUDE_DEBUG_INFO ?>
-    <Media Id="2" Cabinet="runtime_PDBs.cab" EmbedCab="yes" />
-    <?endif?>
-
     <!-- Directory Structure -->
     <Directory Id="TARGETDIR" Name="SourceDir">
       <Directory Id="WINDOWSVOLUME">
         <Directory Id="LIBRARY" Name="Library">
-          <Merge Id="runtime-msm" DiskId="1" Language="1033" SourceFile="$(var.windows-runtime-msm.TargetPath)" />
+          <Directory Id="SWIFT" Name="Swift-development">
```

Comment:
Bleh, this really should get versioned (`Swift-5.2`, `Swift-5.3` and `Swift-development`)